.github/workflows/lint.yaml

```diff
@@ -0,0 +1,34 @@
+name: Lint
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  lint:
+    name: Lint the app
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Setup Ruby 
+      uses: ruby/setup-ruby@v1 # no need for ruby-version because we have a .ruby_version file
+      with:
+          bundler-cache: true
+
+    - name: Cache dependencies
```

Comment:
I wonder if it's even necessary to cache dependencies for a project like this which is built very rarely. (I'm also not sure how long the cache stays in place, and if it's purged automatically or if it costs money to maintain it in Github Actions' cache.)

I'll leave it up to you to decide Ana.

Reply:
I agree, we probably don't need caching on this repo. No one has touched it in ages and afaik there aren't any updates to it happening soon. We can revisit but for now I'll remove caching.